docs/getting-started/integrations/react.md

```diff
@@ -277,6 +277,98 @@ export default App;
 
 For more information, please refer to the {@link getting-started/setup/ui-language Setting the UI language} guide.
 
+### Potential issues with Jest testing
```

Comment:
Usually, I oppose nesting sections, but in some parts of the docs, we have the "Known issues" section. Don't we want to unify this?
```suggestion
## Known issues
### Potential issues with Jest testing
```

Reply:
I renamed it to `Jest testing`, as it is subsection of known issues. wdyt?